docs/src/intro/usage.md

```diff
@@ -94,15 +95,16 @@ As described [here](https://0xpolygonmiden.github.io/miden-vm/intro/overview.htm
 
 After a program finishes executing, the elements that remain on the stack become the outputs of the program, along with the overflow addresses (`overflow_addrs`) that are required to reconstruct the [stack overflow table](../design/stack/main.md#overflow-table).
 
-## Fibonacci example
-In the `miden/examples/fib` directory, we provide a very simple Fibonacci calculator example. This example computes the 1001st term of the Fibonacci sequence. You can execute this example on Miden VM like so:
+## Examples
+### Fibonacci example
+A very simple Fibonacci calculator example, which computes the n-th term of the Fibonacci sequence, could be executed on Miden VM like so:
 ```
-./target/optimized/miden run -a miden/examples/fib/fib.masm -n 1
+./target/optimized/miden example fibonacci -n [index]
 ```
-This will run the example code to completion and will output the top element remaining on the stack.
 
-If you want the output of the program in a file, you can use the `--output` or `-o` flag and specify the path to the output file. For example:
+### BLAKE3 example
+An example, which computes a sequence of hashes using BLAKE3 1-to-1 hash function. The sequence length could be specified with a parameter like so:
 ```
-./target/optimized/miden run -a miden/examples/fib/fib.masm -o fib.out
+./target/optimized/miden example blake3 -n [length]
 ```
-This will dump the output of the program into the `fib.out` file. The output file will contain the state of the stack at the end of the program execution.
+For the convenience of checking the correctness of the hash calculation, the initial value is set as a bit string consisting only of ones.
```

Comment:
I wouldn't change this as it illustrates how to run/prove external MASM files. Let's revert to how it was.